includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -492,6 +492,26 @@ public function is_saved_cards_enabled() {
 		return 'yes' === $this->get_option( 'saved_cards' );
 	}
 
+	/**
+	 * Check if account is eligible for card present.
+	 *
+	 * @param false $empty_value - Default return value.
+	 * @return bool
+	 */
+	public function is_card_present_eligible( $empty_value = false ) {
+
+		if ( ! $this->is_connected() ) {
```

Comment:
Looking into `is_card_present_eligible` implementation, this if-construct looks like a redundant. Can we get rid of it (I also believe we'll also simplify mocking in UTs)?